src/api/app/controllers/webui/project_controller.rb

```diff
@@ -164,7 +164,6 @@ def destroy
   end
 
   def requests
-    @requests = @project.open_requests
```

Comment:
It's weird that this instance variable isn't used, but looking at the views, it's really not needed.

Reply:
Yes. It's probably some remaining code that was not deleted after changing the view to work with AJAX requests.